src/lib.rs

```diff
@@ -427,14 +427,25 @@ pub struct DataFormatDescriptorHeader {
 }
 
 impl DataFormatDescriptorHeader {
```

Comment:
Should we rename this struct to `DataFormatDescriptorBlockHeader`? I think that's what this thing is called in the spec, and distinguishing between the entire DFD _section_ and an individual DFD block helps clarify what all the lengths and offsets mean.

Reply:
Seems reasonable to me.

Reply:
I'd like to move this to a follow-up PR that also improves docs if that's alright.